docs/cart-and-checkout-blocks/README.md

```diff
@@ -0,0 +1,5 @@
+---
+category_title: Cart and checkout blocks 
```

Comment:
We've used title case for the rest of the category names, so probably want to do the same here. `Cart and Checkout Blocks`.